database/sql/getemall_test_data.sql

```diff
@@ -13,5 +13,5 @@ INSERT INTO users (email, salt, password, nickname, verified, role) VALUES
 ;
 
 insert into profiles (id, profile_pic, full_name, pub_email, bio) VALUES
- (2, null, 'Miquifant', 'miqui@personal.fake', 'I\'ll tell you something about me. The thing is that I am miqui. That\'s all I want to say about this particular topic')
+ (2, null, 'Miquifant', 'miqui@personal.fake', 'I''ll tell you something about me. The thing is that I am miqui. That''s all I want to say about this particular topic')
```

Comment:
Did this work?

Reply:
actually... nope.
Tests were failing

Reply:
and shouldn't it be another PR?

Reply:
well,... It is a very small change and it is needed for this issue to work.
If you reject this PR, I will create a new issue, a new PR,... but I think it will be an unnecessary overhead

Reply:
What if you just put it in a separate commit?

Reply:
done